packages/data-explorer-ui/src/components/common/Banner/banner.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Typography } from "@mui/material";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { CallToActionButton } from "../Button/components/CallToActionButton/callToActionButton";
+import { Banner } from "./banner";
+import { BannerPrimary } from "./banner.styles";
+
+export default {
+  argTypes: {
+    children: { control: { disabled: true } },
+  },
+  component: Banner,
+  title: "Components/Common/Banner",
```

Comment:
Let's give the banner the opportunity to take full width of the viewport. Add the following to the story defaults.

```
  parameters: {
    layout: "fullscreen",
  },
```